.github/workflows/core-daily.yaml

```diff
@@ -0,0 +1,311 @@
+# Runs every day. Tests TRC-2 in ALL kernels.
+#
+# TODO: Automate tracee-test-kernels generation through workflows.
+# TODO: Change dockerhub -> github repository once we're sure this is good.
+#
+name: CO-RE (DAILY)
+on:
+  workflow_dispatch: {}
+  schedule:
+    # At 00:00 every day
+    - cron: '0 0 * * *'
+jobs:
+  #
+  # TRC-2 v4.19 (stable)
+  #
+  run-core-test-trc2-v419-stable:
```

Comment:
Similar to my comment in PR validation workflow it's quite easy to use matrix strategy and keep the workflow definition DRY

```yaml
jobs:
  run-core-test:
    runs-on: ubuntu-20.04
    strategy:
      matrix:
        signature:
          - "TRC-2"
        kernel:
          - "4.19.238-stable"
          - "5.4.166-ubuntu+"
          - "5.4.189-stable"
          - "5.8.18-ubuntu"
          - "5.10.111-stable"
          - "5.11.22-ubuntu"
          - "5.11.22-fedora32"
          - "5.13.19-ubuntu"
          - "5.14.18-fedora33"
          - "5.15.30-ubuntu"
          - "5.15.34-stable"
          - "5.16.18-stable"
          - "5.16.19-fedora34"
    steps:
      - name: "Checkout"
        uses: actions/checkout@v2
        with:
          submodules: true
      - name: "Pull test container image"
        run: |
          docker image pull rafaeldtinoco/tracee-test-kernels:latest
      - name: "Run tests"
        run: |
          docker run --rm --privileged -v $(pwd):/tracee:rw \
            -e kvm_accel="tcg" \
            -e kern_version=${{ matrix.kernel }} \
            -e test_name=${{ matrix.signature }} \
            -t rafaeldtinoco/tracee-test-kernels:latest
```